src/ui/public/vislib/visualizations/_map.js

```diff
@@ -275,6 +278,13 @@ export default function MapFactory(Private) {
   TileMapMap.prototype._createMap = function (mapOptions) {
     if (this.map) this.destroy();
 
+    if (this._attr.wms && this._attr.wms.enabled) {
```

Comment:
This `if` statement should just be placed with the one right below doing the exact same check.